src/utils/pdf.test.ts

```
@@ -23,48 +38,590 @@ const generateArrayBuffer = async (data: Uint8Array) => {
+      URL.revokeObjectURL(url);
+      resolve(canvas);
+    };
+
+    img.onerror = () => {
+      URL.revokeObjectURL(url);
+      reject(new Error('Failed to load image'));
+    };
+
+    img.src = url;
+  });
 }
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)